services/web/server/src/simcore_service_webserver/diagnostics_monitoring.py

```diff
@@ -23,22 +23,34 @@
 kCANCEL_COUNT = f"{__name__}.cancel_count"
 
 kCOLLECTOR_REGISTRY = f"{__name__}.collector_registry"
+kPROCESS_POOL_EXECUTOR = f"{__name__}.process_pool_executor.pool"
```

Comment:
hmm this should be removed no? you use a threadpool executor

Reply:
forgot